modules/regreet/nixos.nix

```
@@ -12,7 +12,22 @@
     lib.mkIf
       (config.stylix.enable && config.stylix.targets.regreet.enable && pkgs.stdenv.hostPlatform.isLinux)
       {
+        warnings =
+          let
+            cfg = config.programs.regreet;
+          in
+          lib.mkIf
+            (
+              cfg.regreet.enable
+              &&
+                !config.services.greetd.settings.default_session.command
+                == "${pkgs.dbus}/bin/dbus-run-session ${lib.getExe pkgs.cage} ${lib.escapeShellArgs cfg.cageArgs} -- ${lib.getExe cfg.package}"
```

Provide a link so that we know where the upstream value is defined:

```suggestion
                # TODO: Add permanent link (with HASH), where this commmand was
                # taken from.
                !config.services.greetd.settings.default_session.command
                == "${pkgs.dbus}/bin/dbus-run-session ${lib.getExe pkgs.cage} ${lib.escapeShellArgs cfg.cageArgs} -- ${lib.getExe cfg.package}"
```